CHANGELOG.md

```diff
@@ -1,5 +1,3 @@
-# Changelog
```

Comment:
Shouldn't we also update the other headers as such that they start with `#` instead of `##`?